worlds/cv64/__init__.py

```diff
@@ -0,0 +1,331 @@
+        else:
+            self.active_stage_list = [stage for stage in vanilla_stage_order if stage in self.active_stage_exits]
+
+        # Create a list of warps from the active stage list. They are in a random order by default and will never
+        # include the starting stage.
+        self.active_warp_list = generate_warps(self)
+
+    def create_regions(self) -> None:
+        # Create the Menu region.
+        active_regions = {"Menu": Region("Menu", self.player, self.multiworld)}
+
+        # Create every stage region by checking to see if that stage is active.
+        active_regions.update({name: Region(name, self.player, self.multiworld) for name in
+                               get_region_names(self.active_stage_exits)})
+
+        # Add the Renon's shop region if shopsanity is on.
+        if self.options.shopsanity.value:
+            active_regions.update({rname.renon: Region(rname.renon, self.player, self.multiworld)})
+
+        # Add the Dracula's chamber (the end) region.
+        active_regions.update({rname.ck_drac_chamber: Region(rname.ck_drac_chamber, self.player, self.multiworld)})
+
+        # Add the locations to the regions.
+        for reg_name in active_regions:
+            reg_locs = get_region_info(reg_name, "locations")
+            if reg_locs is not None:
+                active_regions[reg_name].add_locations(verify_locations(self.options, reg_locs), CV64Location)
+
+        # Set up the regions correctly.
+        for region in active_regions:
+            self.multiworld.regions.append(active_regions[region])
+
+        # Add the warp Entrances to Menu.
+        active_regions["Menu"].add_exits(get_warp_entrances(self.active_warp_list))
+
+        # Add the Entrances to the stage regions.
+        for reg_name in active_regions:
+            reg_ents = get_region_info(reg_name, "entrances")
+            if reg_ents is not None:
+                active_regions[reg_name].add_exits(verify_entrances(self.options, reg_ents, self.active_stage_exits))
+
+    def create_item(self, name: str, force_classification=None) -> Item:
+        if force_classification is not None:
+            classification = getattr(ItemClassification, force_classification)
+        else:
+            classification = getattr(ItemClassification, get_item_info(name, "default classification"))
+
+        code = get_item_info(name, "code")
+        if code is not None:
+            code += base_id
+
+        created_item = CV64Item(name, classification, code, self.player)
+
+        return created_item
+
+    def create_items(self) -> None:
+        item_counts = get_item_counts(self)
+
+        # Set up the items correctly
+        for classification in item_counts:
+            for item in item_counts[classification]:
+                self.multiworld.itempool += [self.create_item(item, classification)
+                                             for _ in range(item_counts[classification][item])]
```

Comment:
the way you have it is fine but if you want a funny
```suggestion
        self.multiworld.itempool += [self.create_item(item, classification) for classification in item_counts for item in item_counts[classification] for _ in range(item_counts[classification][item])]
```

Reply:
Sure! XD